spec/models/request_spec.rb

```diff
@@ -244,22 +245,22 @@
       describe '[:counts][:recipients]' do
         subject { stats[:counts][:recipients] }
         it { should eq(8) }
+
+        describe 'messages to us' do
+          before do
+            create(:message, request: request, sender: create(:contributor), broadcasted: true, recipient: nil)
+          end
+
+          it 'are excluded' do
+            should eq(8)
+          end
+        end
       end
 
       describe '[:counts][:photos]' do
         subject { stats[:counts][:photos] }
         it { should eq(10) } # unique photos
       end
-
-      describe 'iterating through a list' do
-        subject { -> { Request.find_each.map(&:stats) } }
-        it { should make_database_queries(count: 35) }
-
-        describe '::include_associations' do
-          subject { -> { Request.include_associations.find_each.map(&:stats) } }
-          it { should make_database_queries(count: 7) } # better
-        end
-      end
```

Comment:
After the changes in this PR, there are actually less db calls without eager loading than with. The last counts from these tests where that the db queries went from the 35 expected to 10 and from 7 with `::include_associations` to 13.
Since using `::include_associations` actually adds three db calls, it has been removed.